src/page-ui.js

```diff
@@ -0,0 +1,224 @@
+import { retrieveProjects } from './localStorage';
+import Todo from './todo';
+import Project from './project';
+
+function createLayout() {
+  const pageContent = document.querySelector('#page-content');
+  const layout = `
+    <section class="hero is-medium has-bg-img">
+      <div class="hero-head">
+        <nav class="navbar">
+          <div class="container">
+            <div id="navbarMenuHeroA" class="navbar-menu">
+              <div class="navbar-end">
+                <span class="navbar-item">
+                  <a id="new-project-button" class="button is-dark is-inverted">
+                    <span class="icon">
+                      <i class="fab fa-github"></i>
+                    </span>
+                    <span>New Project</span>
+                  </a>
+                </span>
+                <span class="navbar-item">
+                  <a id="new-todo-button" class="button is-dark is-inverted">
+                    <span class="icon">
+                      <i class="fab fa-github"></i>
+                    </span>
+                    <span>New Todo</span>
+                  </a>
+                </span>
+              </div>
+            </div>
+          </div>
+        </nav>
+      </div>
+
+      <div class="hero-body">
+        <div class="container has-text-centered">
+          <h1 class="title">
+            Agendum
+          </h1>
+          <h2 class="subtitle">
+            Plan your day...
+          </h2>
+        </div>
+      </div>
+
+      <div class="hero-foot">
+        <div id="form-section" class="container">
+        </div>
+      </div>
+    </section>
+
+    <section class="section">
+      <div class="container">
+        <div class="columns">
+          <div class="column is-3">
+            <aside class="menu">
+              <p class="menu-label">
+                projects
+              </p>
+              <ul id="project-list" class="menu-list">
+              </ul>
+            </aside>
+          </div>
+
+          <div class="column is-9">
+            <div id="todo-list-content"  class="content is-medium">
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>`;
+  pageContent.insertAdjacentHTML('beforeend', layout);
+}
+
+function displayProjects() {
+  const projectList = document.querySelector('#project-list');
+  const projects = retrieveProjects();
+
+  projects.forEach(project => {
+    const { name } = project;
+    const itemContent = `<li><a>${name}</a></li>`;
+    projectList.insertAdjacentHTML('beforeend', itemContent);
+  });
+}
+
+function addNewProjectForm() {
```

Comment:
## Replace function declaration with ES6 syntax
- [ ] Kindly replace this function declaration with the recommended ES6 arrow syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/)) as is has major benefits like allowing us to implicitly return values when there is nobody block, resulting in shorter and simpler code in some cases.